### PR TITLE
flake8

### PR DESCRIPTION
**Pull request by vincentqb**
Addressing some lint, and unused variables.

### CODE REVIEW

File: examples/interactive_asr/vad.py [hunk 1]
```diff
@@ -19,18 +19,16 @@
 
 from collections import deque
 
-import librosa
 import numpy as np
```
Comment (zhangguanheng66) [inline examples/interactive_asr/vad.py]: Do you want the numpy dependency?

Comment (vincentqb) [reply examples/interactive_asr/vad.py]: We use it on line 225, but it'd be nice to get rid of it. I was mostly cleaning lint, but feel free to suggest a replacement :)

Comment (zhangguanheng66) [reply examples/interactive_asr/vad.py]: Sure. It could be solved in a separate PR. Just notice it.

**Comment by vincentqb** [comment]
> why the binary tests fail?

Build with conda has been failing recently. This is something that will have to be looked at.

```
Packaging torchaudio
INFO:conda_build.build:Packaging torchaudio
INFO conda_build.build:build(1570): Packaging torchaudio
Packaging torchaudio-0.4.0.dev20190904-py37
INFO:conda_build.build:Packaging torchaudio-0.4.0.dev20190904-py37
INFO conda_build.build:bundle_conda(891): Packaging torchaudio-0.4.0.dev20190904-py37
compiling .pyc files...
number of files: 32
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): did not find - or even know where to look for: /lib64/libm.so.6
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): Needed DSO lib/libstdc++.so.6 found in [u'libstdcxx-ng']
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): .. but [u'libstdcxx-ng'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): did not find - or even know where to look for: /lib64/libc.so.6
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): did not find - or even know where to look for: /lib64/libpthread.so.0
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): Needed DSO lib/libgcc_s.so.1 found in [u'libgcc-ng']
WARNING (torchaudio,lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so): .. but [u'libgcc-ng'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
Fixing permissions
Packaged license file/s.
WARNING: Detecting which files contain PREFIX is slow, installing ripgrep makes it faster. 'conda install ripgrep'
Detected hard-coded path in binary file lib/python3.7/site-packages/_torch_sox.cpython-37m-x86_64-linux-gnu.so
Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
WARNING:conda_build.build:Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
WARNING conda_build.build:bundle_conda(1042): Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
TEST START: /opt/conda/conda-bld/linux-64/torchaudio-0.4.0.dev20190904-py37.tar.bz2
Adding in variants from /tmp/tmp3ny_rh/info/recipe/conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from /tmp/tmp3ny_rh/info/recipe/conda_build_config.yaml
INFO conda_build.variants:_combine_spec_dictionaries(189): Adding in variants from /tmp/tmp3ny_rh/info/recipe/conda_build_config.yaml
Renaming work directory,  /opt/conda/conda-bld/torchaudio_1567625309188/work  to  /opt/conda/conda-bld/torchaudio_1567625309188/work_moved_torchaudio-0.4.0.dev20190904-py37_linux-64
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed
failed to parse packages from exception: The following specifications were found to be incompatible with each other:

Package more-itertools conflicts for:
pytest -> more-itertools[version='>=4.0|>=4.0,>=4.0,<6.0']
Package sqlite conflicts for:
python[version='>=3.7,<3.8.0a0'] -> sqlite[version='>=3.24.0,<4.0a0|>=3.25.2,<4.0a0|>=3.25.3,<4.0a0|>=3.26.0,<4.0a0|>=3.27.2,<4.0a0|>=3.29.0,<4.0a0']
Package py conflicts for:
pytest -> py[version='>=1.4.33|>=1.5.0']
Package funcsigs conflicts for:
pytest -> funcsigs[version='>=1.0']
Package * conflicts for:
pytorch=1.3.0.dev20190904 -> *[track_features=cpuonly]
scipy -> *[track_features=nomkl]
Package blas conflicts for:
scipy -> blas[version='*|*|1.0',build='openblas|mkl|mkl|openblas']
Package cffi conflicts for:
pytorch=1.3.0.dev20190904 -> cffi
Package openssl conflicts for:
python[version='>=3.7,<3.8.0a0'] -> openssl[version='>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a']
Package packaging conflicts for:
pytest -> packaging
Package attrs conflicts for:
pytest -> attrs[version='>=17.2.0|>=17.4.0']
Package pip conflicts for:
python[version='>=3.7,<3.8.0a0'] -> pip
Package libstdcxx-ng conflicts for:
scipy -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']
python[version='>=3.7,<3.8.0a0'] -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']
Package six conflicts for:
pytest -> six[version='>=1.10.0']
Package ncurses conflicts for:
python[version='>=3.7,<3.8.0a0'] -> ncurses[version='>=6.1,<7.0a0']
Package tk conflicts for:
python[version='>=3.7,<3.8.0a0'] -> tk[version='>=8.6.7,<8.7.0a0|>=8.6.8,<8.7.0a0']
Package typing conflicts for:
torchaudio==0.4.0.dev20190904=py37 -> typing
Package libgfortran-ng conflicts for:
scipy -> libgfortran-ng[version='>=7,<8.0a0|>=7.2.0,<8.0a0']
Package numpy conflicts for:
scipy -> numpy[version='>=1.11.3,<2.0a0|>=1.15.1,<2.0a0|>=1.9.3,<2.0a0']
pytorch=1.3.0.dev20190904 -> numpy[version='>=1.11']
Package mkl-service conflicts for:
scipy -> mkl-service[version='>=2,<3.0a0']
Package pathlib2 conflicts for:
pytest -> pathlib2[version='>=2.2.0']
Package python conflicts for:
pytest -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
typing -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
torchaudio==0.4.0.dev20190904=py37 -> python[version='>=3.7,<3.8.0a0']
scipy -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
pytorch=1.3.0.dev20190904 -> python[version='>=2.7,<2.8.0a0|>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0']
Package mkl conflicts for:
scipy -> mkl[version='>=2018.0.0,<2019.0a0|>=2018.0.2,<2019.0a0|>=2018.0.3,<2019.0a0|>=2019.1,<2020.0a0|>=2019.4,<2020.0a0']
pytorch=1.3.0.dev20190904 -> mkl[version='>=2018']
Package pytorch conflicts for:
torchaudio==0.4.0.dev20190904=py37 -> pytorch==1.3.0.dev20190904
Package libgcc-ng conflicts for:
scipy -> libgcc-ng[version='>=7.2.0|>=7.3.0']
python[version='>=3.7,<3.8.0a0'] -> libgcc-ng[version='>=7.2.0|>=7.3.0']
Package libffi conflicts for:
python[version='>=3.7,<3.8.0a0'] -> libffi[version='>=3.2.1,<4.0a0']
Package ninja conflicts for:
pytorch=1.3.0.dev20190904 -> ninja
Package readline conflicts for:
python[version='>=3.7,<3.8.0a0'] -> readline[version='>=7.0,<8.0a0']
Package pluggy conflicts for:
pytest -> pluggy[version='>=0.12,<1.0|>=0.5,<0.7|>=0.7|>=0.9|>=0.9,!=0.10,<1.0']
Package atomicwrites conflicts for:
pytest -> atomicwrites[version='>=1.0']
Package xz conflicts for:
python[version='>=3.7,<3.8.0a0'] -> xz[version='>=5.2.4,<6.0a0']
Package zlib conflicts for:
python[version='>=3.7,<3.8.0a0'] -> zlib[version='>=1.2.11,<1.3.0a0']
Package libopenblas conflicts for:
scipy -> libopenblas[version='>=0.2.20,<0.2.21.0a0|>=0.3.2,<0.3.3.0a0|>=0.3.3,<1.0a0']
Package wcwidth conflicts for:
pytest -> wcwidth
Package future conflicts for:
pytorch=1.3.0.dev20190904 -> future
Package setuptools conflicts for:
pytest -> setuptools[version='>=40.0']
Package requests conflicts for:
pytorch=1.3.0.dev20190904 -> requests
Package importlib_metadata conflicts for:
pytest -> importlib_metadata[version='>=0.12']
Exited with code 1
```
from [here](https://circleci.com/gh/pytorch/audio/3640?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

**Pull request merged**